hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/config/EvmConfiguration.java

```diff
@@ -67,6 +66,15 @@ CacheManager cacheManagerToken() {
         return caffeineCacheManager;
     }
 
+    @Bean(CACHE_MANAGER_1H)
+    CacheManager cacheManager1H() {
+        final var caffeine =
+                Caffeine.newBuilder().expireAfterWrite(1, TimeUnit.HOURS).maximumSize(1);
+        final CaffeineCacheManager caffeineCacheManager = new CaffeineCacheManager();
```

Comment:
Instead of a hardcoded 1 hr, we have been using a string cache specification for the ones above. We should stick with that here so we can adjust all aspects of the cache via a property. Please also update docs/configuration.md.
```suggestion
    @Bean(CACHE_MANAGER_FEE)
    CacheManager cacheManagerFee() {
        final CaffeineCacheManager caffeineCacheManager = new CaffeineCacheManager();
        caffeineCacheManager.setCacheSpecification(cacheProperties.getFees());
```